packages/storybook/src/schematics/configuration/files/.storybook/webpack.config.js

```diff
@@ -0,0 +1,62 @@
+// Export a function. Accept the base config as the only param.
+module.exports = async ({ config, mode }) => {
+  // `mode` has a value of 'DEVELOPMENT' or 'PRODUCTION'
+  // You can change the configuration based on that.
+  // 'PRODUCTION' is used when building the static version of storybook.
+
+  r = config.module.rules.filter(rule => rule.test != '/\\.s(c|a)ss$/');
+
+  // Make whatever fine-grained changes you need
+  r.push({
+    test: /\.scss$/,
+    // test: /\.s(c|a)ss$/,
+    use: [
+      'to-string-loader',
+      {
+        loader: 'style-loader',
+        options: {
+          sourceMap: true
+        }
+      },
+      {
+        loader: 'css-loader',
+        options: {
+          sourceMap: true
+        }
+      },
+      {
+        loader: 'sass-loader',
+        options: {
+          sourceMap: true
+        }
+      }
+    ]
+  });
+
+  r = r.filter(rule => rule.test != '/\\.css$/');
+
+  // Make whatever fine-grained changes you need
+  r.push({
```

Comment:
Think storybook should match every stylesheet nx supports. Also, do we require custom scss support? Under the hood, it uses angular cli which already has it supported

Reply:
There's some conflict between the angular cli and storybook's webpack config that throws displays an error instead of your storybook content unless we do this custom webpack config workaround.  See: https://github.com/storybookjs/storybook/issues/3593

I haven't tried stylesheets other than css and scss.